packages/shared/src/components/cards/ArticlePostCard.tsx

```diff
@@ -103,6 +106,7 @@ export const ArticlePostCard = forwardRef(function PostCard(
             <PostMetadata
               createdAt={post.createdAt}
               readTime={post.readTime}
```

Comment:
Mentioned in [DR](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/679149569/YouTube+content#Post-card:~:text=duration-,readTime,-video_id) this is the DB field where the duration is going to be saved, so didn't add any field.
Highlighting to make sure.